
From: William Lee Irwin III <wli@holomorphy.com>

in_interrupt() requires hardirq.h; without this reiser4 fails to link.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/reiser4/plugin/cryptcompress.c |    1 +
 1 files changed, 1 insertion(+)

diff -puN fs/reiser4/plugin/cryptcompress.c~reiser4-hardirq-build-fix fs/reiser4/plugin/cryptcompress.c
--- 25/fs/reiser4/plugin/cryptcompress.c~reiser4-hardirq-build-fix	Thu Sep 23 15:03:03 2004
+++ 25-akpm/fs/reiser4/plugin/cryptcompress.c	Thu Sep 23 15:03:03 2004
@@ -43,6 +43,7 @@ The list of cryptcompress specific EA:
 #include <linux/pagemap.h>
 #include <linux/crypto.h>
 #include <linux/swap.h>
+#include <linux/hardirq.h>
 
 int do_readpage_ctail(reiser4_cluster_t *, struct page * page);
 int ctail_read_cluster (reiser4_cluster_t *, struct inode *, int);
_
